### ISSUE TITLE
Can't build cardano-db-sync and db-tool with nix on buildkite CI machine

### ISSUE THREAD

**Issue by ArturWieczorek** [input-output-hk/cardano-node-tests#1818]
I am trying to build `cardano-db-sync` and `cardano-db-tool` in my tests https://buildkite.com/input-output-hk/qa-db/builds/352#01882685-80f4-4ff4-a5f7-759104b1a8fd on Buildkite machine `ci-equinix-6-ci6-1`
using following commands:

`nix-build -v .#cardano-db-sync -o db-sync-node`
`nix-build -v .#cardano-db-tool -o db-sync-tool`

and I am seeing this error:

> error: getting status of '/var/lib/buildkite-agent-iohk/builds/ci-equinix-6-ci6-1/input-output-hk/qa-db/db_sync_tests/cardano-db-sync/.#cardano-db-sync': No such file or directory

> error: getting status of '/var/lib/buildkite-agent-iohk/builds/ci-equinix-6-ci6-1/input-output-hk/qa-db/db_sync_tests/cardano-db-sync/.#cardano-db-tool': No such file or directory

however `cardano-node` and `cardano-cli` were succesfully built with those commands:

```
nix build -v .#cardano-node -o cardano-node-bin
nix build -v .#cardano-cli -o cardano-cli-bin
```

**Comment by ArturWieczorek** [comment]
Could it be similar issue to this ? https://input-output-rnd.slack.com/archives/CC0CLNEH1/p1683593161898199?thread_ts=1682964148.535219&cid=CC0CLNEH1

CC: @jbgi

**Comment by jbgi** [comment]
Should be fixed by 9a4348752ef36e09b5215e174d510dc2ab18b51f: https://buildkite.com/input-output-hk/qa-db/builds/354#018829fa-3b7c-4b95-bff6-72cc65a71153
